### ISSUE TITLE
[Suggestion] All headers ordered - dcrdata.org/proposals

### ISSUE THREAD

**Issue by Michae2xl** [decred/dcrdata#1764]
I would like to see which proposals had a better turnout, plus (vote status and updated) until the date clicking beside (e.g better % first and vice versa)

Link - https://explorer.dcrdata.org/proposals?offset=0&rows=20&byvotestatus=0

![image](https://user-images.githubusercontent.com/44928059/84319854-b1f27380-ab3e-11ea-8e1e-b0d754a7bdff.png)

**Comment by jholdstock** [comment]
It's also a little strange that only one of these headers has a tooltip (Vote Status)

**Comment by ukane-philemon** [comment]
>  I would like to see which proposals had a better turnout, plus (vote status and updated) until the date clicking beside (e.g better % first and vice versa)

Link - https://explorer.dcrdata.org/proposals?offset=0&rows=20&byvotestatus=0

This has already been addressed, pls revisit the link to see the new look. This should mark this closed?

**Comment by jholdstock** [comment]
I disagree. I still see both of the issues described above when visiting that link. It is still not possible to sort the table, and I still see a tooltip on only one of the table headers.

**Comment by chappjc** [comment]
Yeah it's not resolved at all.

**Comment by ukane-philemon** [comment]
> and I still see a tooltip on only one of the table headers.

Vote count and status header both have tooltip, unless we are looking at giving all headers one.
Also I didn't get this clearly

> It is still not possible to sort the table.?

**Comment by chappjc** [comment]
No tooltip is showing for vote count, but I really don't care about the tooltips.

What missing is a possible little up/down icon to change table sorting by a particular column as Michae2xl showed in his painted screenshot.  TBH, that I don't care much about, but that's what this issue is requesting.

**Comment by chappjc** [comment]
A place where column sorting would be more useful is on the transactions tables for mempool and a block, where you might want to sort by DCR amount, fee amount, fee rate, or byte size.

**Comment by jholdstock** [comment]
I think removing all tooltips is more sensible than adding them to all columns

**Comment by ukane-philemon** [comment]
> I disagree. I still see both of the issues described above when visiting that link.

True, I checked dcrdata.decred.org/proposals on my phone when giving feedback on this, there are a lot of differences, two headers with tooltips on mobile but only one on desktop view, also (n% approval) on desktop (n% yes) on mobile and other minor diff.
